records/benchmarks/0.17.3--after-lorem.txt

```diff
@@ -0,0 +1,396 @@
+Running "100 words" suite...
```

Comment:
Ideally I'd have benchmarks comparing before and after in the same benchmark suite, but that's a real schlep to set up.